index.html

```diff
@@ -15,15 +15,22 @@
 <main class="main-container">
     <div class="card">
         <div class="card-image">
-
+            <figure class="Joke">
+                <img class="Petrosyan" src="images/401x512_0xd42ee430_2359835871452856838.png" alt="Я и без того знаю, какой ты Петросян">
+                <figcaption>
+                    Петросян
+                </figcaption>
+                </figure>
```

Comment:
Every tag must be with class. Please change naming as I mentioned below on `<audio>` tag

```Suggestion
Also rename file to smth easier to understand. For expample `card-image-smthaboutpic`
```